### PR TITLE
docker image tagging for v2.1

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
- Update scripts that push docker images to tag with `v2.1` tag instead of `v2`
- Update docker-compose scripts to default to `v2.1` tags instead of `v2`

**Checklist**
- [x] Changes manually tested
- [ ] Automated Tests added/updated
- [x] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docker-compose/cassandra-4.0/README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Both convenience scripts support the following options:
 
 * When using the convenience scripts, the Stargate Docker image (`SGTAG`) used for the `-l` option is the current (snapshot) version as defined in the top level project `pom.xml` file. It can be overridden with the `-t` option on either script:
 
-  `./start_cass_40.sh -t v2.0.0-BETA-2`
+  `./start_cass_40.sh -t v2.0.0`
```
Comment (kathirsvn) [inline docker-compose/cassandra-4.0/README.md:74]: This should be `v2.1.0`?

File: docker-compose/dse-6.8/README.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ Both convenience scripts support the following options:
 
 * When using the convenience scripts, the Stargate Docker image (`SGTAG`) used for the `-l` option is the current (snapshot) version as defined in the top level project `pom.xml` file. It can be overridden with the `-t` option on either script:
 
-  `./start_dse_68.sh -t v2.0.1`
+  `./start_dse_68.sh -t v2.0.0`
```
Comment (kathirsvn) [inline docker-compose/dse-6.8/README.md:78]: This should be v2.1.0?

**Review by kathirsvn — APPROVED** [review]
LGTM except the minor README fix to have `v2.1.0` instead of `v2.0.0`.

File: docker-compose/cassandra-4.0/README.md (hunk 1 above)
Comment (jeffreyscarpenter) [reply docker-compose/cassandra-4.0/README.md:74]: Since it is just an example provided for documentation I don't think it matters what version is used.

**Pull request merged**